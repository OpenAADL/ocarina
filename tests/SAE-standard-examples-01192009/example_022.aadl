-- test 8.7 Bus Components Access draft-01192009

package Example2

-- add for test
public
  with Hardware;
-- end add

system simple
end simple;
system implementation simple.impl
subcomponents
A: processor Hardware::PPC;
B: device Equipment::DigCamera;
connections
bus access A.USB1 <-> B.USB2;
end simple.impl;
processor PPC
features
USB1: provides bus access Hardware::USB;
end PPC;
device DigCamera

-- bad syntax
--   properties
-- correction
features
USB2: requires bus access USB;
end DigCamera;
end Example2;