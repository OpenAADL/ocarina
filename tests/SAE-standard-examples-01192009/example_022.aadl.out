example_022.aadl:7:08: Hardware (identifier) is not a package or a property set visible or existing
Cannot analyze AADL specifications
